slurm/nccl-test-distributed-gb200-nvl72-enroot.slurm

```diff
@@ -59,18 +59,17 @@ fi
 echo "Using nodes: $SLURM_JOB_NODELIST"
 
 # When launching a pyxis job as a non-root user you need to use --no-container-remap-root and when launching as root
-# you need to use --container-remap-root. This is becuse the container is built with Ubuntu 22.04+ and is built to use PMIx. 
+# you need to use --container-remap-root. This is because the container is built with Ubuntu 22.04+ and is built to use PMIx.
 
-if [ $(whoami) = "root" ]; then
+if [ "$(whoami)" = "root" ]; then
   cflag="--container-remap-root"
-else 
+else
   cflag="--no-container-remap-root"
 fi
 
 # The srun command inherits the SBATCH flags set above
 # and will be launched once for each task.
 
 srun --container-image="$CONTAINER_IMAGE" \
-     --mpi=pmix "$cflag" --kill-on-bad-exit=1 --container-mount-home \
+     --mpi=pmix "$cflag" --no-container-mount-home --kill-on-bad-exit=1 \
```

Comment:
This was the only file among all of these that used `--container-mount-home`, which is the default anyway, while the others use `--no-container-mount-home`, so I'm assuming this was a mistake, hence this change.